rules/successful-reachouts/rule.md

```diff
@@ -0,0 +1,37 @@
+---
+type: rule
+title: Do you know to make your Linkedln reachouts stand out?
+uri: successful-reachouts
+authors:
+  - title: Brady Stroud
+    url: https://www.ssw.com.au/people/brady-stroud
+created: 2024-03-1T01:55:35.941Z
+guid: 012a5f87-5cfb-4002-8d5f-4d2a8787ed6a
+---
+
+Developers receive a lot of reachouts. Sometimes they are automated and other times they are a person copying and pasting the same message to everyone.
+
+To have your reachout rise above the noise, you need to make it personal and unique.
+
+![Figure: Typical developers LinkedIn inbox](developer-reachout-spam.png)
+
+<!-- endintro -->
+
+Here are some tips to make your reachouts stand out:
+
+1. Use emojis - a few emojis can make your message stand out, and feel more personal.
+2. Ask a question - most reachouts don't have a proper call to action, they simply ask you to "get in touch". Asking a question engages the candidate and makes them more likely to respond.  
+E.g. If you see someone is learning Rust, you could ask "How long have you been learning Rust? We have a few "Rustaceans" in the SSW Brisbane office 🦀"  
+Its important to ask the question early in the message, as most people will only glance at the .
```

Comment:
Hey @bradystroud 

- [ ] 1. Fix from "Its" to "It's "
- [ ] 2. I think there is a word missing at the end "glance at the ."
- [ ] 3. Remove the full stops from the lists as per https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists